tests/scss/SCSSLintBearTest.py

```diff
@@ -20,7 +20,7 @@
 bad_file2 = '''
 $value: 5px;
 
-.foo {
+ .foo {
```

Comment:
I think that there is no need to add a space here.

Reply:
It's an instance of a bad file. So I added an extra space to test that the linter indeed returns a number message or warning.

Reply:
Sorry, my bad